test/ui/BasePage.ts

```diff
@@ -13,6 +13,8 @@ export default class BasePage {
   }
 
   async enterTextById(id: string, text: string) {
+    const element = await this.driver.findElement(By.id(id));
```

Comment:
Do we use this element value?

Reply:
Yes, we're using this to capture the element as a constant in order to implement the clear method below it (line 17). This removes any text inside the element prior to sending keys. Currently it's being used to clear any existing email and password info in the login form. Is this OK?

Reply:
Can it be chained? `await this.driver.findElement(By.id(id)).clear()` If not that's ok as is

Reply:
Removed constant & consolidated method.